packages/api/src/application/features/assessments/getList/controller.ts

```diff
@@ -12,7 +12,12 @@ export class GetAssessmentListController extends BaseController {
     super();
   }
 
-  protected async executeImpl(req: Request): Promise<Assessment[]> {
-    return Promise.reject(new Error(`Not implemented`));
+  protected async executeImpl(req: Request, res: Response): Promise<void> {
+    try {
+      const assessments = await this.getAssessmentListUseCase.execute();
+      res.status(200).json({ assessments });
```

Comment:
```suggestion
      return assessments 
```
Because of `BaseController` which this class extends, you only need to return the data. You don't have to handle the response.